ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-dependencies/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[hclecomm.dependencies]"
+    embed="[hclecomm.vendor]"/>
```

Comment:
@DeepaliAgrwl : You would need  to checkin css.txt and js.txt within the clientlib dependencies. 
else clientlib dependencies wont load

 
![image](https://user-images.githubusercontent.com/38690824/82201742-72a97c00-991e-11ea-87be-73b7ad2986c0.png)